ding/model/wrapper/model_wrappers.py

```diff
@@ -866,10 +866,13 @@ def forward(self, *args, **kwargs):
         assert isinstance(output, dict), "model output must be dict, but find {}".format(type(output))
         if 'action' in output or 'action_args' in output:
             key = 'action' if 'action' in output else 'action_args'
-            action = output[key]
+            action = output[key]['action_args'] if isinstance(output[key], dict)  else output[key]
```

Comment:
Hello, thank you for your contribution. There should be one space before else, and a comment needs to be added to explain that this is done to ensure compatibility with hybrid action spaces. Once these changes are made, this PR can be merged. Thanks!

Reply:
Updated.